layouts/partials/ecosystem/registry/entry.html

```diff
@@ -86,37 +86,48 @@
     {{ end -}}
     {{ $primaryHref := printf "href=%q" $primaryUrl | safeHTMLAttr -}}
     <li class="card my-3 registry-entry {{ $highlightStyle }}" data-registry-id="{{ ._key }}" data-registrytype="{{ .registryType }}" data-registrylanguage="{{ .language }}">
-        <div class="card-body container">
-        <h4 class="card-title mb-0 d-flex flex-row">
-            <a {{ $primaryHref }} target="_blank" rel="noopener" class="me-auto">
-            {{- .title | markdownify -}}
-            </a>
-            <div class="ms-auto">
-            {{ if $isNew -}}
-            <span class="badge rounded-pill text-bg-info"><i class="fa-regular fa-star"></i> new</span>
-            {{ end -}}
-            {{ if $isNative -}}
-            <span class="badge rounded-pill text-bg-success text-white"><i class="fa-solid fa-puzzle-piece"></i> native</span>
-            {{ end -}}
-            {{ if $isFirstParty -}}
-            <span class="badge rounded-pill text-bg-success text-white"><i class="fa-solid fa-heart"></i> first party integration</span>
-            {{ end -}}
-            {{ if $usedInDemo -}}
-            <span class="badge rounded-pill text-bg-secondary text-white" title="This package is used in the OpenTelemetry Demo!"><i class="fa-solid fa-shapes"></i> OTel Demo</span>
-            {{ end -}}
-            {{ if $deprecated -}}
-            <span class="badge rounded-pill text-bg-danger text-white" title=""><i class="fa-solid fa-ban"></i> deprecated</span>
-            {{ end -}}
-            {{ if .cncfProjectLevel -}}
-            <span class="badge rounded-pill text-bg-primary text-white" title="CNCF {{ .cncfProjectLevel}} Project">
-                <img alt="CNCF" style="display: inline-block; width: 14px; height: 14px; border: none; margin:0px; padding: 0; vertical-align:middle" src="/img/cncf-icon-white.svg"> {{ .cncfProjectLevel }}
-            </span>
-            {{ end -}}
-            </div>
+      <div class="card-body container">
+      <h4 class="card-title mb-0 d-flex flex-row">
+          <a {{ $primaryHref }} target="_blank" rel="noopener" class="me-auto">
+          {{- .title | markdownify -}}
+          </a>
+          <div class="ms-auto">
+          {{ if $isNew -}}
+          <span class="badge rounded-pill text-bg-info"><i class="fa-regular fa-star"></i> new</span>
+          {{ end -}}
+
+          {{ if $isNative -}}
+          <button type="button" class="badge rounded-pill text-bg-success text-white"
+          onclick="document.getElementById('input-s').value = 'native'; document.getElementById('input-language').value = 'all';document.getElementById('input-component').value = 'all';document.getElementById('searchForm').submit();">
```

Comment:
Can you move the javascript code into a dedicated file? Since it is related to it, you can put it into https://github.com/open-telemetry/opentelemetry.io/blob/main/assets/js/registrySearch.js

I suggest that you add some `data-`-tag to the elements and then select them in javascript:

```
<button type=button" data-filter-value="native" class="...">...native</button>
```

and then something like

```js
document.querySelectorAll('[data-filter-value]').forEach(element => ...)
```

Also, provide a tooltip or something that says "click to filter by..."

Reply:
I appreciate your feedback. I will update the code accordingly.

Reply:
Hello @svrnm  I’ve implemented the recommended changes. However, I observed an issue: after clicking a tag and the results page is rendered, clicking the tag again or another doesn’t trigger a new search. The reset button has to be clicked before the tag filter can work again.

<img width="1275" alt="Screenshot 2024-10-13 at 1 37 18 AM" src="https://github.com/user-attachments/assets/0b2e1875-dedc-49b2-a8ca-e16f7cc8612f">

Reply:
@basiratkareem can you append the label to the existing search term instead of setting/replacing it. This way someone can have an existing search word and refine their search with adding one of the labels. The only thing you might want to check for if the label is already included in the search string so it's not added twice

Reply:
Hi @svrnm I tried implementing your suggestions however the behaviour was the same.(see screenshot below) I then realised that when I run the npm server with the command `npm run serve`, the issue exists however running the server with command `npm run serve:netlify` the labels work as expected. clicking on the label at any point initiates a search.

Reply:
Here is the link to the screen recording demonstrating the details mentioned in my previous comment. [https://www.loom.com/share/428fef104ab04901ad73cf6cea7c18ac?sid=3007315f-df87-459b-82db-fa4d8a8f10f4](url)

Reply:
I look forward to your feedback. Thank you.

Reply:
It's still not working, please take a look in the auto generated preview:

https://deploy-preview-5340--opentelemetry.netlify.app/ecosystem/registry/

If I click on a button (e.g. native of the first element) the search is updated, but if I click another button it's not updated once again

Reply:
Hi @svrnm,

Thank you for your feedback. I have taken a look at the preview link and that is the same experience on my end when I use the command `npm run serve`

However, It worked on my end using `npm run serve:netlify`, but I see there’s still an issue in the Netlify preview. I’ll check the code again and I’ll update you soon. Thank you.